kars/0001-accelerator-driver-runtime-management/README.md

```diff
@@ -0,0 +1,44 @@
+# KAR-0001: Accelerator Driver & Runtime Management
+
+## Description
+
+Provide a verifiable mechanism for ensuring that compatible accelerator drivers and corresponding container runtime configurations are correctly installed and maintained on nodes with accelerators.
```

Comment:
Thinking ahead about automated tests, it will be hard to provide tests for all the different types of accelerators. Would the automated test default to the most common accelerator? What if a platform does not support the accelerator expected by the automated tests? How would they meet this requirement?